website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -625,6 +666,25 @@ spec:
         },
       ],
     },
+    {
+      name: 'Destination',
+      type: 'DestinationConfig: <optional>',
+      description: `Controls configuration specific to proxies to destinations through terminating-gateway. Added in v1.13.0.`,
+      children: [
+        {
+          name: 'Addresses',
+          type: 'array<string>: []',
+          description: `List of addresses associated with the destination. Could be a hostname or an IP`,
+          yaml: false,
```

Comment:
Since this is part of the Kube CRD, I think we need that field.

```suggestion
          description: `List of addresses associated with the destination. Could be a hostname or an IP. Wildcards are not accepted.`,
          yaml: `List of addresses associated with the destination. Could be a hostname or an IP. Wildcards are not accepted.`,
```